### PR TITLE
[LIB-196] quick fix for social auth login

### PR DESCRIPTION
**Pull request by opalczynski**

### CODE REVIEW

**Comment by dancio** [comment]
@opalczynski please fix tests

**Comment by opalczynski** [comment]
@dancio fixin fixin :)

File: syncano/connection.py [hunk 1]
```diff
@@ -349,10 +348,11 @@ def authenticate_admin(self, **kwargs):
             if self.is_social:
                 request_args = self.validate_params(kwargs,
                                                     self.SOCIAL_LOGIN_PARAMS)
+                request_args['access_token'] = request_args.pop('token')  # core expects a access_token field;
             else:
                 request_args = self.validate_params(kwargs,
                                                     self.LOGIN_PARAMS)
-
+        print(request_args)
```
Comment (Zhebr) [inline syncano/connection.py]: remember to remove this

File: syncano/connection.py [hunk 2]
```diff
@@ -383,6 +383,7 @@ def __init__(self, *args, **kwargs):
     @property
     def connection(self):
         # Sometimes someone will not use super
+        from syncano.models.registry import registry
```
Comment (23doors) [inline syncano/connection.py]: Btw, why move it here? As it was working before without it and it doesn't seem like other changes made it necessary.

**Pull request merged**